bfcfg

```diff
@@ -27,16 +27,26 @@
 # of the authors and should not be interpreted as representing official policies,
 # either expressed or implied, of the FreeBSD Project.
 
-bfcfg_version=0.1
+bfcfg_version=0.2
 cfg_file=/etc/bf.cfg
 log_file=/tmp/bfcfg.log
 dump_mode=0
 
+efivars=/sys/firmware/efi/efivars
+efi_global_var_guid=8be4df61-93ca-11d2-aa0d-00e098032b8c
+efi_vlan_var_guid=9e23d768-d2f3-4366-9fc3-3a7aba864374
+rshim_mac_sysfs=${efivars}/RshimMacAddr-${efi_global_var_guid}
```

Comment:
Perhaps, we should use "rshim_efi_mac_sysfs" to be consistent with "oob_efi_mac_sysfs"?

Reply:
done